### PR TITLE
New Cel2 Version of Docs

### PR DESCRIPTION
**Pull request by sodofi**

### CODE REVIEW

File: docs/developer/explorers/block-explorers.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Block Explorer
+---
+import ColoredText from '/src/components/ColoredText';
+
+
+
+## [Blockscout](https://celo.blockscout.com)
+
+Blockscout explorer is available for <ColoredText>[Celo](https://explorer.celo.org/mainnet/blocks)</ColoredText>. If your contract is verfied on Blockscout, you can interact with and debug smart contracts.
```
Comment (lvpeschke) [inline docs/developer/explorers/block-explorers.md:10]:
```suggestion
Blockscout explorer is available for <ColoredText>[Celo](https://celo.blockscout.com)</ColoredText>. If your contract is verfied on Blockscout, you can interact with and debug smart contracts.
```

**Review by lvpeschke — APPROVED** [review]

**Comment by lvpeschke** [comment]
Note: I did not review every page, relying on the other reviewers to have a look here.

File: docs/build/add-celo-testnet-to-metamask.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Add Celo Testnet to Metamask
+---
+
+Follow this quick guide to add Dango to your MetaMask wallet and start playing around with it. You can get funds from the [Alfajores Faucet](https://faucet.celo.org).
```
Comment (lvpeschke) [inline docs/build/add-celo-testnet-to-metamask.md:5]:
```suggestion
Follow this quick guide to add Alfajores to your MetaMask wallet and start playing around with it. You can get funds from the [Alfajores Faucet](https://faucet.celo.org).
```

File: docs/build/add-celo-testnet-to-metamask.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+If you never added a new Network before then you need click on "Add a Network"
+:::
+
+Insert the following details:
+
+- Network Name: Celo Alfajores Testnet
+- New RPC URL: `https://alfajores-forno.celo-testnet.org`
+- Chain ID: 44787
+- Currency Symbol: CELO
+- Block explorer URL: `https://explorer.celo.org/alfajores`
```
Comment (aaronmgdr) [inline docs/build/add-celo-testnet-to-metamask.md:31]:
```suggestion
- Block explorer URL: `https://celo-alfajores.blockscout.com`

File: docs/build/build-on-minipay/code-library.mdx [hunk 4]
```diff
@@ -0,0 +1,620 @@
...
+
+        ```bash
+        yarn add @celo/abis @celo/identity viem@1
+        ```
+
+  </TabItem>
+  <TabItem value="npm" label="npm" attributes={{ className: "npm_tab" }}>
+
+        ```bash
+        npm install @celo/abis @celo/identity viem@1
```
Comment (aaronmgdr) [inline docs/build/build-on-minipay/code-library.mdx:80]:
```suggestion
        npm install @celo/abis @celo/identity viem@2

File: docs/build/how-to-deploy/verify/celo-explorer.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: Verify with Celo Explorer
```
Comment (aaronmgdr) [inline docs/build/how-to-deploy/verify/celo-explorer.md:2]: this should be replace with blockscout verifiucation page.

File: docs/build/integrations/guide.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+## Wallet Integration
+
+These guidelines apply to any application that manages keys and allows users to interact and transfer value on the Celo platform.
+
+### Key Derivation
+
+Celo wallets should follow [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) for deriving private keys from [BIP39 mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). Celo's key derivation path is `m/44'/52752'/0'/0`. The first key is typically the `account key` for balance transfers, while the second key can be derived as the `dataEncryptionKey` for encrypting information.
+
+### Identity Protocol
```
Comment (aaronmgdr) [inline docs/build/integrations/guide.md:50]: call this Social Connect

File: docs/build/integrations/guide.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+Celo has a [lightweight identity protocol](/protocol/identity) that allows users to address each other via phone numbers. To protect user privacy, Celo wallets should implement the [Phone Number Privacy protocol](/protocol/identity/odis-use-case-phone-number-privacy) to prevent large-scale harvesting of phone numbers.
+
+### Wallet Address Management
+
+When transferring assets, wallets should check the receiving account's `walletAddress`. Smart contract accounts may have different recovery characteristics but receive funds at a different address. A `walletAddress` of `0x0` indicates that a different mechanism is required to acquire the `walletAddress`.
```
Comment (aaronmgdr) [inline docs/build/integrations/guide.md:56]: i dont think this is really true

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
45 pages and not even a description. can we do better?

**Comment by aaronmgdr** [comment]
remember that sweet bridging page @therealharpaljadeja made for the dango launch? lets bring that back

File: docs/build/integrations/guide.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+The `ReleaseGold` smart contract, which is audited, allows for the scheduled release of CELO to users.
+
+## Wallet Integration
+
+These guidelines apply to any application that manages keys and allows users to interact and transfer value on the Celo platform.
+
+### Key Derivation
+
+Celo wallets should follow [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) for deriving private keys from [BIP39 mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). Celo's key derivation path is `m/44'/52752'/0'/0`. The first key is typically the `account key` for balance transfers, while the second key can be derived as the `dataEncryptionKey` for encrypting information.
```
Comment (aaronmgdr) [inline docs/build/integrations/guide.md:48]: note to self about derivation key standard. re defaulting to ethereum

**Pull request closed without merging**